src/index.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
 import { render } from 'react-dom';
 import App from './components/App/App';
+import { Provider } from 'react-redux';
+import { store } from './store/store';
```

Comment:
can we create an index file to avoid such paths like `/store/store` with duplicated namings